app/templates/layouts/_survey.html

```diff
@@ -7,7 +7,7 @@
   </div>
   <div class="bar__inner container">
     <div class="badge badge--amber u-mr-xs">BETA</div>
-    <div class="bar__title">{{ bar_title }}</div>
+    <div class="bar__title">{% if meta %}{{ meta.survey.title }}{% endif %}</div>
```

Comment:
Question for @hamishtaplin does this div always need to be present? It is empty if the survey has no title, can we just remove it from the DOM if no title is present?